Ginger/Ginger/MainWindow.xaml.cs

```diff
@@ -145,6 +151,10 @@ public void Init()
             finally
             {
                 HideSplash();
+                if (autoLoadSolDone == false && WorkSpace.Instance.Solution == null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4555652)